.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,19 @@
+# Sync GitHub issues to Jira issues
+
+# Configuration syntax:
+# https://github.com/canonical/gh-jira-sync-bot?tab=readme-ov-file#client-side-configuration
```

Comment:
Unsure if this should be a permalink or point to the README in case that heading gets reworded.